### DISCUSSION TITLE
Consider moving `spec/` to `doc/spec/`

### DISCUSSION THREAD

**Discussion by ghost** [badges/shields#8782] [Ideas] [answered]
When I first noticed the `spec/` directory, I thought it was a test directory 😅 This may be some test runners rubbing off on me, but I also think that it's also an opportunity to consolidate the documentation. `doc/spec/` is just a suggestion. Another option could be `doc/specification/`.

**Comment by krishna9786** [comment]
No problem! It's understandable that you initially mistook the "spec/" directory for a test directory, especially if you're used to working with test runners. Your suggestion to use "doc/spec/" or "doc/specification/" as an alternative is a great idea. It would indeed help consolidate the documentation related to specifications and make it clearer for other developers.

**Comment by PyvesB** [comment, marked as answer]
Thanks for raising this. I do agree with the observation. However, there are likely a number of place on the Internet pointing to our specification in its current location; after all it's been there for a decade. This would essentially be a breaking change for them, and I personally don't think the benefits would be worth it.
